src/lib/tls/tls_channel.cpp

```diff
@@ -104,6 +104,18 @@ void Channel::reset_active_association_state()
       m_sequence_numbers->reset();
    }
 
+void Channel::set_prestate(uint16_t in_message_seq, uint16_t out_message_seq)
```

Comment:
I think these should be uint64_t since theoretically the initial client hello could have been retransmitted and lost many times before we got it and sent the hello verify request.

Reply:
16-bit message sequence here, not the 48-bit record sequence number. It seems that we don't need to take care of record sequence since it only affects decryption? Correct me if I am wrong.